docs/user-interface/skills_knowledge_contributions.md

```diff
@@ -0,0 +1,83 @@
+---
+title: InstructLab UI Skills & Knowledge contributions
+description: Steps to contribute to the community model via UI.
+logo: images/ilab_dog.png
+---
+
+The UI Simplifies the process for Skills & Knowledge contributions by:
+
+* Minimising risk of human error when writing YAML by using the web form. 
+
+* Directly submit a github pull request with a press of a button.
+
+When the form is filled out, you also are given the option to download the YAML and attribution files to your local machine, and to view the form in its original YAML structure before submission.
+
+You can view all your submissions on the dashboard page.
+
+!!! warning
+    Even when running the UI locally, you must be logged in via github to successfully submit your Knowledge and Skills contributions. You can still fill out the form, and download the YAML and attribution files.
+
+For tips on writing Skills & Knowledge contributions, please visit the documentation under the [Taxonomy](/taxonomy/) heading.
+
+## Skill Contributions
+
+### Author Information
+
+Enter your github email address, and your full name, which will be tied to your contribution.
+
+### Skill Information
+
+Give a brief description on what skill you would like to teach.
+
+### Taxonomy Directory Path
+
+Specify where in the taxonomy this skill would belong in. As the taxonomy fills, it will become more complex, make sure to give a concise location to keep the taxonomy organised.
```

Comment:
```suggestion
Specify where in the taxonomy repository directory structure this skill would belong in. As the taxonomy fills in with more skills, it will become more complex, make sure to give a concise location to keep the taxonomy organised. 
```